<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemInfoForm</class>
 <widget class="QFrame" name="SystemInfoForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>506</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>280</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#SystemInfoForm{
border-radius:8px;
background-color: #F9FAFF;
background-image: url(:/images/system-info-back.svg);
background-repeat: none;
border: 1px solid rgba(213,223,238, 0.6);
}</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>25</number>
   </property>
   <property name="topMargin">
    <number>28</number>
   </property>
   <property name="rightMargin">
    <number>25</number>
   </property>
   <item>
    <widget class="QLabel" name="labelSystemInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>27</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel#labelSystemInfo{
color: #1B1B24;
font: 20px;
font-weight: 500;
background-image: url(:/images/system-info.svg);
background-position: left;
background-repeat: none;
background-color: rgba(0, 0, 0, 0);
padding-left: 30;
}
</string>
     </property>
     <property name="midLineWidth">
      <number>0</number>
     </property>
     <property name="text">
      <string>系统信息</string>
     </property>
     <property name="textFormat">
      <enum>Qt::PlainText</enum>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QWidget" name="lineWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:#646DA3;
}</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <property name="topMargin">
      <number>19</number>
     </property>
     <property name="bottomMargin">
      <number>12</number>
     </property>
     <item>
      <widget class="QLabel" name="labelPlan">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelPlan{
background-color: rgba(0,0,0,0);
color: #3A3D48;
font-weight: 400;
font-size: 16px;
} </string>
       </property>
       <property name="text">
        <string>当前配置方案:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelPlanText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelPlanText{
background-color: rgba(0,0,0,0);
color: #1B1B24;
font-weight: 500;
font-size: 16px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="lineMiddle" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0.1);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <property name="topMargin">
      <number>13</number>
     </property>
     <property name="bottomMargin">
      <number>13</number>
     </property>
     <item>
      <widget class="QLabel" name="labelProductID">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>22</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelProductID{
background-color: rgba(0,0,0,0);
color: #3A3D48;
font-weight: 400;
font-size: 16px;
}</string>
       </property>
       <property name="text">
        <string>当前批次样品序号:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelProductIDText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>21</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelProductIDText{
color: #1B1B24;
font-weight: 500;
background-color: rgba(0, 0, 0, 0);
font-size: 16px;
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="lineMiddle_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>1</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgba(0, 0, 0, 0.1);</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>12</number>
     </property>
     <property name="topMargin">
      <number>15</number>
     </property>
     <item>
      <widget class="QLabel" name="labelProgressText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel#labelProgressText{
background-color: rgba(0,0,0,0);
}</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;
                    &lt;span style=&quot;font-size:16px; font-weight:400;&quot;&gt;等待启动：第&lt;/span&gt;
                    &lt;span style=&quot;color:#3B5AFD;font-size:17px; font-weight:500;&quot;&gt;0&lt;/span&gt;
                    &lt;span style=&quot;font-size:16px; font-weight:400;&quot;&gt;块，共&lt;/span&gt;
                    &lt;span sytle=&quot;color:#1B1B24;font-size:17px; font-weight:500;&quot;&gt;0&lt;/span&gt;
                    &lt;span style=&quot;font-size:16px; font-weight:400;&quot;&gt;块&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>12</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>12</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar::chunk {
background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #3B5AFD,stop:1 #00C6F1);
border-radius:0px;
width: 1px;
}

QProgressBar#progressBar
{
background: #E2E6FF;
border: 1px solid rgba(255, 255, 255, 0);
}</string>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_3">
     <property name="topMargin">
      <number>42</number>
     </property>
     <item row="0" column="0">
      <widget class="QPushButton" name="buttonStop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonStop{
background-color:white;
background-image: url(:/images/stop-button.svg);
background-repeat: none;
background-position:center;
border-radius: 24px;
border: 0px solid;
}

QPushButton#buttonStop:hover{
background-color: #3B5AFD;
background-image: url(:/images/stop-button-white.svg);
}

QPushButton#buttonStop:disabled{
background-color: #FFFFFF;
background-image: url(:/images/stop-button-grey.svg);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="buttonPlay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>68</width>
         <height>68</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonPlay{
background-color: white;
background-image: url(:/images/play-button.svg);
background-repeat: none;
background-position: center;
border-radius: 34px;
border: 0px solid;
}

QPushButton#buttonPlay:hover{
	background-color:#3B5AFD;
	background-image: url(:/images/play-button-white.svg);
}

QPushButton#buttonPlay:disabled{
	background-color:#3B5AFD;
	background-image: url(:/images/play-button-white.svg);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="buttonProduct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton#buttonProduct{
border-radius: 24px;
border: 0px solid;
background-color:white;
background-image: url(:/images/product-shift.svg);
background-repeat: none;
background-position:center;
}

QPushButton#buttonProduct:hover{
	background-color:#3B5AFD;
	background-image: url(:/images/product-shift-white.svg);
}

QPushButton#buttonProduct:disabled{
background-color: #FFFFFF;
background-image: url(:/images/product-shift-grey.svg);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <property name="topMargin">
      <number>10</number>
     </property>
     <property name="horizontalSpacing">
      <number>20</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="labelPlay">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>43</horstretch>
         <verstretch>27</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color:rgba(0, 0, 0, 0);
color: #101820;
font-size: 16px;
font-weight: 400;
}</string>
       </property>
       <property name="text">
        <string>启动</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelStop">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>67</horstretch>
         <verstretch>22</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color:rgba(0, 0, 0, 0);
color: rgba(0, 0, 0, 0.3);
font-size: 16px;
font-weight: 400;
}</string>
       </property>
       <property name="text">
        <string>停止运行</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="labelProduct">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>67</horstretch>
         <verstretch>22</verstretch>
        </sizepolicy>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel{
background-color:rgba(0, 0, 0, 0);
color: #101820;
font-size: 16px;
font-weight: 400;
}</string>
       </property>
       <property name="text">
        <string>产品管理</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
